### PR TITLE
deposit: panel icon support

### PR DESCRIPTION
**Pull request by jmartinm**
Signed-off-by: Javier Martin Montull <javier.martin.montull@cern.ch>

### CODE REVIEW

**Comment by jmartinm** [comment]
Allows to specify an icon in the deposit panels 
screen shot 2015-08-19 at 3 40 57 pm

File: invenio_deposit/form.py [hunk 1]
```diff
@@ -24,7 +24,8 @@
 CFG_GROUPS_META = {
     'classes': None,
     'indication': None,
-    'description': None
+    'description': None,
+    'icon': None
```
Comment (jirikuncar) [inline invenio_deposit/form.py]: @jmartinm can you end the line with comma (`,`)?

**Comment by jirikuncar** [comment]
Please add `NEW` or `BETTER` label to commit message explaining how to use this feature. Thanks

**Comment by jmartinm** [comment]
Updated.

**Comment by jirikuncar** [comment]
:shipit:

**Comment by jirikuncar** [comment]
@jmartinm can you pls rebase? Thanks

**Comment by jmartinm** [comment]
Rebase.

**Pull request merged**